app/models/completed_review_turnaround.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: completed_review_turnarounds
+#
+#  id                :bigint           not null, primary key
+#  value             :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  review_request_id :bigint           not null
+#
+# Indexes
+#
+#  index_completed_review_turnarounds_on_review_request_id  (review_request_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (review_request_id => review_requests.id)
+#
+
+class CompletedReviewTurnaround < ApplicationRecord
+  belongs_to :review_request
```

Comment:
why are we storing the reference to only one review request?

Reply:
i think this was the simplest association because we can navigate through all the info from here, it would be harder to collect information about the review if we store the reference of a PR or something like that